### PR TITLE
Memory Usage config updates #163738628

### PR DESCRIPTION
**Pull request by LoganArnett**
This PR is for [ticket #163738628](https://www.pivotaltracker.com/story/show/163738628). This ticket was for investigating the memory usage escalating in Production. 

While running the app locally in either development or production mode I was able to see a stable memory usage of roughly 300mb:
image

This PR also includes the `oj` gem for Optimized JSON parsing

### CODE REVIEW

File: config/scout_apm.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This configuration file is used for Scout APM.
+# Environment variables can also be used to configure Scout. See our help docs at http://docs.scoutapm.com#environment-variables for more information.
+common: &defaults
+
+  # key: Your Organization key for Scout APM. Found on the settings screen.
+  # - Default: none
+  key: s4ehrWvArTiWqSpyvLHZ
```
Comment (akegan) [inline config/scout_apm.yml]: is this OK to be public?

File: config/puma.rb [hunk 2]
```diff
@@ -53,6 +55,11 @@
 #   ActiveRecord::Base.establish_connection if defined?(ActiveRecord)
 # end
 #
+# before_fork do
```
Comment (akegan) [inline config/puma.rb]: is this here so we can un-comment if we decide to use it?

**Review by akegan — COMMENTED** [review]
just a couple of questions, otherwise looks good.

File: config/scout_apm.yml (hunk 1 above)
Comment (LoganArnett) [reply config/scout_apm.yml]: That is a good question, it should most likely be moved to an ENV, and I assume that you will want to use the key for your account as the main one

File: config/puma.rb (hunk 2 above)
Comment (LoganArnett) [reply config/puma.rb]: removed this for now

File: config/scout_apm.yml (hunk 1 above)
Comment (LoganArnett) [reply config/scout_apm.yml]: moved to `SCOUT_KEY='SCOUT_KEY_HERE'`

**Review by akegan — APPROVED** [review]
looks good

**Pull request merged**